src/Components/CanvasComponent.tsx

```diff
@@ -0,0 +1,243 @@
+import React from "react";
+import { SketchPicker } from "react-color";
+
+interface Point {
+  x: number;
+  y: number;
+}
+
+interface Color {
+  hex: string;
+}
+
+class CanvasComponent extends React.Component {
+  state = {
+    background: "#121212",
+    choice: "brush",
+    canvas: {} as HTMLCanvasElement,
+  };
+
+  handleChangeComplete = (color: any) => {
+    this.setState({ background: color.hex });
+  };
+
+  componentDidMount() {
+    this.updateCanvas();
+  }
+  updateCanvas() {
+    var isDrawing: boolean, lastPoint: Point;
+    var canvas: HTMLCanvasElement = document.getElementById(
+      "canvas"
+    ) as HTMLCanvasElement;
+    var canvasWidth: number = canvas.width;
+    var canvasHeight: number = canvas.height;
+    var ctx: any = canvas.getContext("2d");
+    var jobStart: boolean = false;
```

Comment:
First try to use `const`. If you feel the need to change the variable, use `let`. `var` should not be used at all in most cases, because of its weird scope.